test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1392,7 +1413,7 @@ describe('getUserParticipantFromConversationEvent()', () => {
   });
 
   it('should return void if no update', () => {
-    expect(handler.getUserParticipantFromConversationEvent(undefined)).toBe(undefined);
+    expect(handler.getUserParticipantFromConversationEvent(undefined as any)).toBe(undefined);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(handler.getUserParticipantFromConversationEvent(undefined as unknown)).toBe(undefined);
```